### PR TITLE
Fix httpclient fork in managed-services

### PR DESCRIPTION
**Pull request by gsaslis**
**What this PR does / why we need it**:

* Updates the fork used for the `httpclient` ruby gem

same exact code on both forks - the 3scale one having the benefit that it doesn't depend on an external (nowadays) contributor.

* Fixes version of httpclient

the `require_relative` change is necessary for bundler to be able to correctly resolve the dependencies provided by Cachito

(cherry picked from commit f916a10ef03068a119f84f394730e098e24ba6f7)
> Replace this with a quick explanation of what's included here. 

**Special notes for your reviewer**:

I would like to promote this fix to `managed-services` stream, so that continuous product builds are functional again. (they'll be broken until this is ported over). 

Perhaps merging all commits from `master` to `managed-services` is preferred, but in case it isn't, we can use this PR in the meantime.

### CODE REVIEW

**Review by akostadinov — CHANGES_REQUESTED** [review]
we should rather push master to managed-services. We can do that any time. Oterwise we break fast-forward between the branches.

**Comment by gsaslis** [comment]
@akostadinov up to you ! 

In this specific case, it's true that a merge would cause a merge conflict but it is a very very straightforward one to resolve, right? If we are ready to merge `master` to `managed-services`, then it would be great if we could do that, otherwise can we merge this in the meantime to unblock the cachito migration ?

**Comment by jlledom** [comment]
I prefer to merge this into `master` and then FF master into `managed-servides`. We got a [github action](https://github.com/3scale/porta/actions/workflows/fast-forward-managed-services.yml) that performs the fast-forward automatically and it can't fix conflicts no matter how easy they are. After merging this to `master` I'll FF to `managed-services` manually myself.

**Comment by gsaslis** [comment]
@jlledom it's already in `master` so fast-forwarding would be great ! ;)

**Comment by gsaslis** [comment]
once you do that, please feel free to close this PR ;)

**Comment by jlledom** [comment]
Done!

**Pull request closed without merging**
